js/index.js

```diff
@@ -1,6 +1,44 @@
 // Iteration 1: Names and Input
-//
+const hacker1 = "Saskia"
+console.log(`The driver's name is ${hacker1}`)
+const hacker2 = "Martin"
+console.log(`The navigator's name is ${hacker2}`)
 // Iteration 2: Conditionals
-
+if(hacker1.length > hacker2.length){
+    console.log('The driver has the longest name, it has XX characters.')
+} if (hacker2.length > hacker1.length){
+    console.log('It seems that the navigator has the longest name, it has XX characters.')
+} else if (hacker2.length === hacker1.length){
+    console.log('Wow, you both have equally long names, 6 characters!')
```

Comment:
In those `console.log` you could have used the `<variable>.length` so it would adapt to different scenarios